### ISSUE TITLE
Außentemperatur -5572,77 C

### ISSUE THREAD

**Issue by fred0815** [SBorg2014/WLAN-Wetterstation#76]
![image](https://github.com/user-attachments/assets/ebaf2e63-98e8-4092-b328-6ba419cceb1b)
![image](https://github.com/user-attachments/assets/7fc77d07-b55f-4637-b332-f9ae3fb3a5ee)

seems to be a currupted value at Außentemperatur, Taupunkt and Luftfeuchtigkeit.

running cuurent version of 3.2.0

**Comment by SBorg2014** [comment]
Yes, most of the Stations/Gateways send sometimes a "NULL" (nothing, not "0" value) instead of a proper value. The result of the calculation with this "NULL" value is than always -5572.77 °C.
Change in `wetterstation.conf` line #107 (**_FIX_AUSSENTEMP_**) from "false" to "true" (don't forget to restart service `sudo systemctl restart wetterstation`).
 > #Fix aktivieren bei fehlerhafter Außentemperatur [true/false] / default: false
 > #Bei unplausiblem Messwert wird kein Datenpaket an den ioB geschickt
 > FIX_AUSSENTEMP=false
   
This fixed not the error from the Station/Gateway, but dropped the wrong measurement and don't send it to the ioB.

**Comment by fred0815** [comment]
Thanks a lot!
